scripts/setup/README.md

```diff
@@ -1,17 +1,9 @@
-## Getting Started
+# Getting started
 
 ### Windows
+1. Open up a Terminal.
+2. Type this command: `git clone https://github.com/fosscord/fosscord && .\scripts\setup\setup.bat`.
 
-to setup all repositories open cmd and enter this to command
-
-```
-curl https://raw.githubusercontent.com/fosscord/fosscord/main/scripts/setup/setup.bat --output setup.bat && setup.bat
```

Comment:
I think curl is better, because It'll only download the setup file and setup alle repositories in the root folder not in a sub dir of the overview repo

Reply:
Also curl is installed per default in windows 10